index.src.html

```diff
@@ -695,15 +695,7 @@ <h4 id="algorithm-request" algorithm>Request a `Credential`</h4>
 
     2.  Assert: |settings| is a [=secure context=].
 
-    3.  Return [=a promise rejected with=] `NotSupportedError` if any of the following statements
-        are true:
-
-        1.  |settings| does not have a [=environment settings object/responsible document=].
-
-        2.  |settings|' [=environment settings object/responsible document=] is not the
```

Comment:
This check is now moved into the specific implementation that are executed `[=in parallel=]`. Is that ok? There is a pending pull request to limit access to the `[=environment settings object=]` in parallel execution.

Reply:
Hrm. I think you're right. We're now calling `current settings object`, which I guess is difficult to get once we're inside the parallel world. That's annoying.

I suppose we could pass in an "is same-origin with the top-level document" flag instead. That's a little annoying, but whatever.

Reply:
@equalsJeffH Do you agree with this proposal? It would be a change to WebAuthN but would not have any impact on the implementations, I guess.

Reply:
The change to WebAuthN would basically be accepting a new boolean and throwing it away. Seems reasonable enough to me, if it enables behavior that WebAuthN values.